types/luxon/src/datetime.d.ts

```diff
@@ -304,6 +305,8 @@ export interface LocaleOptions {
     locale?: string | undefined;
     outputCalendar?: CalendarSystem | undefined;
     numberingSystem?: NumberingSystem | undefined;
+    weekSettings?: WeekSettings | undefined;
+    defaultToEN?: boolean | undefined;
```

Comment:
```suggestion
```
I don't think this is meant to be exposed. It doesn't look like this being input will have any effect, based on what I see from src. Do you have an example/use case?

Reply:
I saw this in the source code, but then reviewed the docs and it's not in the docs. `weekSettings` is there, so I'll keep that one.